### PR TITLE
Fixed #25869 -- Added set_trim and set_precision methods to WKTWriter.

### PR DESCRIPTION
**Pull request by sir-sigurd**
https://code.djangoproject.com/ticket/25869

### CODE REVIEW

File: docs/ref/contrib/gis/geos.txt [hunk 1]
```diff
@@ -1052,6 +1052,44 @@ Returns the WKT of the given geometry. Example::
     >>> wkt_w.write(pnt)
     'POINT (1.0000000000000000 1.0000000000000000)'
 
+.. attribute:: WKTWriter.trim
+
+.. versionadded:: 1.10
+
+This property could be used to enable/disable trimming of
```
Comment (timgraham) [inline docs/ref/contrib/gis/geos.txt]: could be -> is? "enable or disable"

File: docs/ref/contrib/gis/geos.txt [hunk 2]
```diff
@@ -1052,6 +1052,44 @@ Returns the WKT of the given geometry. Example::
...
+    'POINT (1.0000000000000000 1.0000000000000000)'
+    >>> wkt_w.trim = True
+    >>> wkt_w.write(pnt)
+    'POINT (1 1)'
+
+.. attribute:: WKTWriter.precision
+
+.. versionadded:: 1.10
+
+This property controls the rounding precision of coordinates,
```
Comment (timgraham) [inline docs/ref/contrib/gis/geos.txt]: , -> ; or separate sentence

File: docs/releases/1.10.txt [hunk 3]
```diff
@@ -94,6 +94,11 @@ Minor features
 
 * Added support for instantiating empty GEOS geometries.
 
+* The new :attr:`~django.contrib.gis.geos.WKTWriter.trim` and
+  :attr:`~django.contrib.gis.geos.WKTWriter.precision` properties
+  of :class:`~django.contrib.gis.geos.WKTWriter`
+  allow to control output of fractional part of coordinates in WKT.
```
Comment (timgraham) [inline docs/releases/1.10.txt]: allow controlling output of the fractional part of the coordinates

**Comment by sir-sigurd** [comment]
@timgraham, thanks for review, updated.

**Comment by claudep** [comment]
Great, committed in c984e2bc15

**Pull request closed without merging**